test/methods/test_presentation_style.ts

```
@@ -101,3 +101,50 @@ test("defaultSpeaker error no speaker", async () => {
     MulmoPresentationStyleMethods.getDefaultSpeaker(presentationStyle);
   });
 });
+
+// --- getResolvedSlideTheme (mulmoclaude#1622 follow-up) ---
+//
+// The priority surfaced here is the single source of truth that both
+// the renderer (`slide.ts`) and the editor (`@mulmocast/deck-web`)
+// will read from, so the tests below pin the contract loudly.
+
+const fakeTheme = (label: string) => ({
+  colors: { bg: label, bgCard: label, bgCardAlt: label, text: label, textMuted: label, textDim: label, primary: label, accent: label, success: label, warning: label, danger: label, info: label, highlight: label },
```

ESLint

[error] Replace `·bg:·label,·bgCard:·label,·bgCardAlt:·label,·text:·label,·textMuted:·label,·textDim:·label,·primary:·label,·accent:·label,·success:·label,·warning:·label,·danger:·label,·info:·label,·highlight:·label` with `⏎····bg:·label,⏎····bgCard:·label,⏎····bgCardAlt:·label,⏎····text:·label,⏎····textMuted:·label,⏎····textDim:·label,⏎····primary:·label,⏎····accent:·label,⏎····success:·label,⏎····warning:·label,⏎····danger:·label,⏎····info:·label,⏎····highlight:·label,⏎·`

(prettier/prettier)